src/bp/ui-admin/src/Pages/Workspace/Bots.jsx

```diff
@@ -99,14 +99,12 @@ class Bots extends Component {
     )
   }
 
-  // TODO implement this properly
   async requestPromotion(botId) {
-    const { data } = await api.getSecured({ timeout: 10000 })({
+    // TODO implement this properly
+    await api.getSecured({ timeout: 10000 })({
       method: 'post',
-      url: `/admin/bots/${botId}/promote`
+      url: `/admin/bots/${botId}/stage`
```

Comment:
not important, but theres a shorthand: getSecured().post()

Reply:
not actual implementation